bitwise.js

```diff
@@ -0,0 +1,12 @@
+
+function getMaxLessThanK(n, k) {
```

Comment:
you have to define type of n , k because now we can also pass undefined , string etc. in this function